src/pages/collective.js

```diff
@@ -60,6 +61,7 @@ class CollectivePage extends React.Component {
     const { LoggedInUser } = this.state;
 
     if (!data.Collective) {
+      ssrNotFoundError(data);
```

Comment:
Can we make the `ErrorPage` component handle this? Or does it need to be a separate page?

Reply:
NextJS requires us to define a `_error.js` file in the pages folder. See https://nextjs.org/docs/#custom-error-handling